x/foundation/codec/cdc.go

```diff
@@ -0,0 +1,18 @@
+package codec
```

Comment:
https://github.com/cosmos/cosmos-sdk/pull/16062
This file isn't needed. Amino codec is deprecated.